packages/@okta/vuepress-site/docs/guides/registration-inline-hook/prepare-your-external-service/nodejs/prepare.md

```diff
@@ -0,0 +1,31 @@
+In this implementation of an external service, the code is written for Node and uses the Express framework. Several npm packages are required:
+
+```javascript
+const express = require('express');
+const bodyParser = require('body-parser');
+```
+
+The following code listens for HTTPS requests:
+
+```javascript
+const app = express();
+const port = process.env.PORT || 3000;
+```
+
+```javascript
+app.listen(port, () => console.log(`Listening on port ${port}!`));
+```
+
```

Comment:
Is this necessary when using Glitch? Wouldn't Glitch listen for requests on its own by default?

Reply:
I'm confused by this whole section. Is the Glitch app and-running-somewhere to try?

Reply:
Glitch includes this code to listen for request, by default, in its application, although I'd need to doublecheck this. The point of this section is to make sure users have dependent code, outside parsing the hook call, included in the app, as well as basic info on what this content is doing. Providing a glitch project, which could be commented, might address that here. Here is the Glitch project I used: https://glitch.com/~aged-garrulous-mango . Content is very close to what could be included for users.